docs/enhancements/using-dtk-for-building-driver-containers.md

```diff
@@ -93,8 +93,8 @@ If the user want us to pull the correct DTK image on builds/cluster-upgrade, he
 
 #### Dockerfile which let KMMO find the correct DTK image.
 ```
-ARG DTK-AUTO
-FROM ${DTK-AUTO}
+ARG DTK_AUTO
```

Comment:
are we going to support DTK only as argument to Dockerfile, or also as a template in the dockerfile that is going to be replaced ( just as template strings in images?

Reply:
What would be the benefit of a template? I would stick to the build arg, which is what a Dockerfile writer would expect.

Reply:
It looks clean to use the build arg as you can inspect the buildConfig, see that `DTK_AUTO` specified by the user and we only added a build arg (instead of manipulating the user's Dockerfile).

Closing this thread.